indico/modules/events/sessions/templates/management/sessions/session_principals.html

```diff
@@ -0,0 +1,13 @@
+{% from 'forms/_form.html' import simple_form %}
+
+{% block content -%}
+    <div class="i-box">
+        <div class="i-box-content">
+            <ul class="group-list no-content-before">
+            {% for principal in session.acl_entries -%}
+                {{ principal }}
```

Comment:
i know it's still WIP but we need to show their roles (manager if full_access, conveneer if `coordinate` roles)